docusaurus/docs/Android/04-compose/02-component-architecture.mdx

```diff
@@ -143,7 +151,13 @@ override fun onCreate(savedInstanceState: Bundle?) {
 
 As you can see, these components are still fairly easy to use, but do require instructions on what state they show and how they look. This snippet will produce the following UI:
 
-![The SearchInput](../assets/compose_default_search_input_component.png)
+| Search Input (Light Theme) |
```

Comment:
It's a very wide yet short components so I decided to put it  top and bottom vs side by side